examples/website/i3s/components/toggle-switch.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import styled from "styled-components";
+
+const Input = styled.input`
+  height: 0;
+  width: 0;
+  opacity: 0;
+  z-index: -1;
+`;
+
+const Label = styled.label`
+  position: absolute;
+  display: inline-block;
+  margin-top: 2px;
+  margin-left: 180px;
+  font-size: 6px;
+  width: 4.8em;
+  height: 1.5em;
+  cursor: ${props => (props.disabled ? "not-allowed" : "pointer")};
+  ${Input} {
+    opacity: 0;
+    width: 0;
+    height: 0;
+  }
+`;
+
+const Slider = styled.span`
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgb(36, 39, 48);
+  -webkit-transition: 0.4s;
```

Comment:
if you want to use browser-specific extensions for `transition`, use it everywhere in your code.